Smith_Ben/Assignments/Login_Registration/server.py

```diff
@@ -0,0 +1,98 @@
+from flask import Flask, redirect, request, render_template, session, flash
+from mysqlconnection import MySQLConnector
+from flask.ext.bcrypt import Bcrypt
+import re
+app = Flask(__name__)
+bcrypt = Bcrypt(app)
+mysql = MySQLConnector(app, 'logindb')
+app.secret_key = 'secret'
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+
+@app.route('/')
+def index():
+	# email = request.form['regemail']
+	# password = request.form['regpwd']
+	# query = "SELECT * FROM users WHERE email = :email LIMIT 1"
+	# data = { 'email': email }
+	# user = mysql.query_db(query, data)
+	# if bcrypt.check_password_hash(user[0][pw_hash], password):
+	# 	return redirect ('/success')
+	# else:
+	# 	flash('Invalid Username or Password')
+	# 	return redirect ('/')
+
+	return render_template('index.html')
+
+@app.route('/create', methods=['POST'])
```

Comment:
why call it `create` when this is actually `login`? You will be confused by your own code when you come back to look at it, as will other developers if methods aren't aptly named.